src/components/AddNewBook.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AddNewBook({ handleSubmit }) {
+  return (
+    <div className="newTask-container">
+      <h2>ADD NEW BOOK</h2>
+      <form onSubmit={handleSubmit}>
+        <input type="text" name="addNew" placeholder="Book Title" />
+        <select name="category">
+          <option value="" defaultValue>Category</option>
+          <option value="Adventure">Adventure</option>
+          <option value="Science Fiction">Science Fiction</option>
+          <option value="Terror">Terror</option>
+          <option value="Romance">Romance</option>
+          <option value="Economy">Economy</option>
+          <option value="Technology">Technology</option>
+        </select>
+        <button type="submit">ADD BOOK</button>
```

Comment:
- [ ] (OPTIONAL) This is not really optional. In the add book form, you are to have inputs for the title and author of a book and not the title and category. Kindly rectify this issue 👍 
- Requirements:
![Annotation 2022-06-03 051845](https://user-images.githubusercontent.com/85113567/171785880-aa6062e5-d834-46f0-8a4c-b07e1b816770.jpg)